newspaper/source.py

```diff
@@ -204,21 +232,22 @@ def parse_categories(self):
 
         self.categories = [c for c in self.categories if c.doc is not None]
 
+    def _map_title_to_feed(self, feed):
```

Comment:
This is a really interesting interpretation of feed parsing! I'll give it a pass for now and do some testing on it later to see how it performs.